identity-apps-core/apps/authentication-portal/src/main/webapp/emailOtpError.jsp

```diff
@@ -64,6 +65,8 @@
                 if (unlockTime != null) {
                     errorMessage = String.format(AuthenticationEndpointUtil.i18n(resourceBundle, "error.user.account.temporarly.locked"), unlockTime);
                 }
+            } else if (errorMessage.equalsIgnoreCase(EmailOTPAuthenticatorConstants.ERROR_TENANT_MISMATCH_MSG)) {
+                errorMessage = "Application you are trying to access does not allow users from your organization.";
```

Comment:
will this be translated properly to the UI when we use different language?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4112/commits/135d38558b9b9ddc8aa32e359f9519ee36f7aa43